### PR TITLE
feat: goerli payments

### PR DESCRIPTION
**Pull request by romtref**
## Description of the changes
Deploy and verify goerli contracts

### CODE REVIEW

File: packages/smart-contracts/src/lib/artifacts/Erc20SwapConversion/index.ts [hunk 1]
```diff
@@ -18,6 +18,10 @@ export const erc20SwapConversionArtifact = new ContractArtifact<ERC20SwapToConve
           address: '0x1d6B06C6f7adFd9314BD4C58a6D306261113a1D4',
           creationBlockNumber: 9447192,
         },
+        goerli: {
```
Comment (yomarion) [inline packages/smart-contracts/src/lib/artifacts/Erc20SwapConversion/index.ts:21]: You can keep the 2nd one only. Cf. the end of the file, the default version is `0.2.0`

**Review by yomarion — APPROVED** [review]
Good !

File: packages/smart-contracts/src/lib/artifacts/ERC20EscrowToPay/index.ts [hunk 2]
```diff
@@ -21,6 +21,10 @@ export const erc20EscrowToPayArtifact = new ContractArtifact<ERC20EscrowToPay>(
           address: '0xEbe28A2B7336670Ba752bfEad4a121D2c4FF2464',
           creationBlockNumber: 10461945,
         },
+        goerli: {
```
Comment (benjlevesque) [inline packages/smart-contracts/src/lib/artifacts/ERC20EscrowToPay/index.ts:24]: ⚠️ there is no contract at that address
https://goerli.etherscan.io/address/0xEbe28A2B7336670Ba752bfEad4a121D2c4FF2464#code

**Review by benjlevesque — COMMENTED** [review]
looks good appart from 1 missing contract

FYI I did not review the scritpts-create2 part

**Review by rom1trt — COMMENTED** [review]
seems good

**Review by KolevDarko — APPROVED** [review]

**Comment by rom1trt** [comment]
move to #891

**Comment by rom1trt** [comment]
move to #891

**Pull request closed without merging**